pygeos/_geometry.pyx

```diff
@@ -109,7 +111,7 @@ def simple_geometries_1d(object coordinates, object indices, int geometry_type,
     if np.any(indices[1:] < indices[:indices.shape[0] - 1]):
         raise ValueError("The indices must be sorted.")  
 
-    cdef double[:, :] coord_view = coordinates
+    cdef double[:] coord_view = coordinates.ravel()
```

Comment:
We would need to ensure the correct layout here (row major (XYXY), i.e. C contiguous). I suppose this is simply `.ravel(order="C")`, although that might actually already be the default.